docs/mobility/slam/_index.md

```diff
@@ -22,14 +22,20 @@ Breaking changes are likely to occur, and occur often.
 SLAM is an important area of ongoing research in robotics, particularly for mobile applications such as drones, boats, and rovers.
 
 The Viam SLAM service supports the integration of SLAM as a service on your robot.
-You can conduct SLAM with data collected live by a [RPlidar](/registry/examples/rplidar/) or with LIDAR data you provide in configuration, and easily view the map you build on the **SLAM library** tab of your location's page in the [Viam app](https://app.viam.com):
+You can conduct SLAM with data collected live by a [RPlidar](https://app.viam.com/module/viam/rplidar) or with LIDAR data you provide in configuration, and easily view the map you build on the **SLAM library** tab of your location's page in the [Viam app](https://app.viam.com):
 
 ![Completed SLAM maps in the SLAM library tab](/mobility/slam/view-map-page.png)
 
 ## Used With
 
 {{< cards >}}
-{{< relatedcard link="/build/configure/components/camera/" alt_title="RPlidar" required="yes" alt_link="/registry/examples/rplidar/">}}
+{{< relatedcard link="ml/vision/vision-odometry/"  required="yes" alt_link="https://app.viam.com/module/viam/rplidar">}}
+{{< relatedcard link="/build/configure/components/camera/" alt_title="RPlidar" required="yes" alt_link="https://app.viam.com/module/viam/rplidar">}}
```

Comment:
@npentrel Took your rec for using external url with `alt_link` here but it is giving me a markdownlint error: no bare urls used

Reply:
Interesting. I don't know a way around it so let's turn off that rule. What's your thinking asking using app module URLs vs The github repo?

Reply:
I am still wondering why module URLs vs github repo - Andrew asked this too on the PR.

Reply:
Also `{{< relatedcard link="ml/vision/vision-odometry/"  required="yes" alt_link="https://app.viam.com/module/viam/rplidar">}}` is not showing up. The link is not where that model lives.

Reply:
@npentrel What do you mean/what can I do to turn off that rule?

Reply:
Also @npentrel I am going to link that card to GitHub as well as thatmodel page will be removed.

Reply:
Wait, isn't this part inaccurate? In my understanding it wouldn't be Rplidar or Cartographer / Camera, it would be Rplidar / Cartographer. Want to confirm with you but should I change? @npentrel

Reply:
I don't know a way to do this without triggering the markdownlint error. You can turn off the markdownlint rule in the .markdownlint settings. But it would be better to turn the rule off for this file only. Please check the markdownlint docs on how to do this. I don't know off the top of my head

Whether the module docs are on the docs site does not affect whether it's required for slam. If it is a requirement you should link to it.


@andf-viam said this about the two options in a previous comment. Probably worth asking the SME to confirm